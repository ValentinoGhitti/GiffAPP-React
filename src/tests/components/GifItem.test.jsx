import { render } from "@testing-library/react";
import { GifItem } from "../../components";

describe('Tests del GifItem component', () => { 
  
  const title = 'One Piece';
  const url = 'https://mugiwara.com.ar/sanji.jpg';


  test('Tiene que hacer MATCH con el Snapshot', () => {
    const {container} = render(<GifItem title={title} url={url} />);
    expect(container).toMatchSnapshot();
  });

});