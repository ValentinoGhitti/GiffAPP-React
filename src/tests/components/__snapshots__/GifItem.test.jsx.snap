// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests del GifItem component Tiene que hacer MATCH con el Snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="One Piece"
      src="https://mugiwara.com.ar/sanji.jpg"
    />
    <p>
      One Piece
    </p>
  </div>
</div>
`;
